application/src/assets/locales/ru/translation.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Login": "Логин",
+  "Cars": "Автомобиль",
+  "Profile": "Профиль",
+  "Registration": "Регистрация",
+  "Card": "Платежная карта",
+  "Email": "Электронная почта",
+  "Password": "Пароль",
```

Comment:
```
{
"common": {
  "buttons": {
    "save": "Save changes",
    "close": "Close"
  }
},
"cart": {
  ""
},
"header": {
  "links": {
     "cars": "Cars",
     "cart": "Cart"
  }
}
}
```


`common.buttons.cancel`
`t('common.buttons.cancel')`


JSON rules

Structure for RU and EN - should be same!